src/js/helpers/Util.js

```diff
@@ -132,6 +132,22 @@ var Util = {
       b != null &&
       a.length === b.length &&
       a.every((v, i) => v === b[i]);
+  },
+  compareProperties: function (a, b, ...keys) {
+    return keys.reduce((memo, key) => {
+      var aVal = a[key];
+      var bVal = b[key];
+      var typeOfA = typeof(aVal);
```

Comment:
Yes, thanks (I had an isObject check and then realised it was already implemented)